specification/keyvault/resource-manager/readme.azureresourceschema.md

```diff
@@ -54,7 +54,6 @@ output-folder: $(azureresourceschema-folder)/schemas
 input-file:
   - Microsoft.KeyVault/preview/2018-02-14-preview/keyvault.json
   - Microsoft.KeyVault/preview/2018-02-14-preview/providers.json
-  - Microsoft.KeyVault/preview/2018-02-14-preview/secrets.json
```

Comment:
Creating secrets in an ARM template is supported. Why is this being removed?

/cc @jlichwa

Reply:
Same question. It is supported.

Reply:
Is it supported in SDK? It's removed because there's no reference in https://github.com/Azure/azure-rest-api-specs/blob/9350ca1cb722e775b057c08adb80959b4bfd618f/specification/keyvault/resource-manager/readme.md#tag-package-2018-02-14-preview

Reply:
I've added it to the readme.md tag, and re-synced the schema readme file. 6b4afb3

Reply:
@jlichwa even with the changes @leni-msft made, I'm still concerned that _secrets.json_ appears to be missing from a lot of tags - even ones where the newer _keys.json_ was added. Is there someone that can verify it's present in all configurations where it makes sense.

Reply:
@leni-msft it does not matter if SDK is using it (secrets.json is purely for ARM templates) and this is supported https://docs.microsoft.com/en-us/azure/templates/microsoft.keyvault/2018-02-14/vaults/secrets . I have no idea who maintains those tags, if there are mostly for SDK... secrets.json is supported since 2016 and have to exist in all versions since then.

Reply:
@jlichwa The readme.md tags are mainly for generating management plane SDKs.  By `secrets.json is purely for ARM templates`, do you mean it's possible that the secrets.json is only supported by ARM templates, but not by management SDKs?
I realized the secrets.json reference was added manually with https://github.com/Azure/azure-rest-api-specs/pull/11964 by @anthony-c-martin . @anthony-c-martin any insights?
Let me revert the changes of KV's schema.md made in this PR.

Reply:
@leni-msft it is the intent to be used purely by ARM templates as stated in docs : https://docs.microsoft.com/en-us/azure/templates/microsoft.keyvault/vaults/secrets 
"...as APIs using this model are is intended for internal use in ARM deployments. Users should use the data-plane REST service for interaction with vault secrets."
For interaction with secrets with SDKs data plane should be used. We also don't provide any public documentation (https://docs.microsoft.com/en-us/rest/api/keyvault/) for secrets on management plane for the same reason.
My assumption is that ARM templates requires swagger and was added manually for ARM templates api, since they need it to support creating secrets through ARM templates which calls management plane.

Reply:
Thanks @jlichwa . It's clear to me now. KeyVault secrets.json is an exception case. I've reverted the changes made to KeyVault in this PR.